### ISSUE TITLE
[BUG]New synced full node, no pools showing  

### ISSUE THREAD

**Issue by Anturaju93** [Chia-Network/chia-blockchain#7947]
**Describe the bug**
After installation of Chia blockchain v.1.2.0 or 1.2.3 pools not showing under Pool tab. I have only one option to create a new NFT, even if I already joined a pool on another computer or install. I'm using the same private key and full synced.   

**Steps to reproduce the behavior:**
1. Reinstall Chia blockchain on another device or after computer was formatted. 
2. Use the same private key.
3. Copy the db mainnet and wallet files to the new install and use same private key, to avoid syncing from 0.
4. Wait to sync 100% mainnet and walled (all green).
5. Go to Pool tab -> no pools present.   

**Expected behavior**

After sync is complete, the Pool tab should display pools I have joined in the past.  

**Quick fix**
I join another pool, create a plot NFT, Self pool, click create and after 2 minutes the old pools are showing and pooling. The problem is that creating another NFT will be added to my private key forever. Now I have 3 NFTs, 2 self pooling and one farming.   

- OS: Windows 10 and Raspberry OS 64

**Additional context**
I tried adding the pool information to the config file, but Chia blockchain was not connecting to wallet. 
After I created another self pooling NFT, spending 1 mojo, because the new install was not showing my pool, now I see all of my NFTs on my second full node running on my plotting computer. 
Some solution must be implemented to show pools NFT after install on different device without the need to create a new NFT, or resync from 0.

**Comment by msn1983aa** [comment]
And we need a quit/delete pool/self pooling function

**Comment by tangdasan** [comment]
Similar problems, the plots number is different after synchronization

**Comment by Anturaju93** [comment]
> And we need a quit/delete pool/self pooling function

I understand that deletion is impossible, but a show/hide or ignore function in the GUI will help a lot.

**Comment by ByronAP** [comment]
I think if you don't copy over the wallet file and just import the mnemonics you wont have a problem because the chain will be scanned and a new wallet file created.

**Comment by Anturaju93** [comment]
My farm is running on raspberry pi and I already had a full synced node on another computer, using the same private key so it makes sense not to wait days to resync on the pi. After I copy over the db files everything works fine and synced 100% but the config file does not get updated with pools list info. Only after I join another pool (create a new NFT and spend 1 mojo) the config file gets updated and all previous pools appear in the pool tab.

**Comment by Gammaubl** [comment]
I’m having this issue as well and I don’t wanna create another plotNFT. Any solutions found?

**Comment by Gammaubl** [comment]
Hey Anturaju93,
It seems like I have found a solution. The problem seems to come from the wallet file that you had copied from another machine. Basically, your wallet file has synced higher than the point that the plotNFTs were created. Therefore, the GUI didn’t know that there are plotNFTs and that’s why you didn’t see them in the GUI.

The solution is that you will have to delete the wallet file (only wallet not db), then wait for it to get re-synced again. I have encountered the same problem as yours when I set up the Raspberry Pi 4 as my second harvester. Luckily for me, I have a backup file of db and wallet from May, which is before the point of plotNFT creation, so I doesn't have to re-synced again from the beginning.

Hope this helps and happy farming!
